src/transactions/lockup.ts

```diff
@@ -43,7 +43,7 @@ const buildCoinLockupMetadata = (params: CoinLockupRequestParams) => {
   );
   // TODO: make sure this replace is correct.
   metadata.lockupAmountBaseUnits = hexToBytes(
-    params.LockupAmountBaseUnits.replace('0x', 'x')
+    stripHexPrefix(params.LockupAmountBaseUnits)
```

Comment:
@stas-kh - can you search for other usages of `.replace('0x', 'x')` and swap in the `stripHexPrefix` function please?